### PR TITLE
Add GH temp token action

### PR DESCRIPTION
**Pull request by bittelc**
Create action which generates a temporary access token for GitHub Actions. This will future Actions which call this Action and use the temporary token output to then trigger other GitHub Actions workflows.

### CODE REVIEW

**Comment by bittelc** [comment]
@pekala – Prelim review. I'll get with you shortly to discuss how to js.

**Comment by bittelc** [comment]
Ha, @pastjean I meant to tag you in this, but when you type `pa` it assumes @pekala, not you.
Sorry Maciek!

File: github-temp-token/src/main.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const getToken = (previews?: string[]) => {
+  const githubAppID = +core.getInput("github_app_id");
+  const githubAppInstallationID = +core.getInput("github_app_installation_id");
+  const githubAppPrivateKey = core.getInput("github_app_private_key");
+
+  const app = new App({ id: githubAppID, privateKey: githubAppPrivateKey });
+  const installationAccessToken = await app.getInstallationAccessToken({
+    installationId: githubAppInstallationID
+  });
+  return `token ${installationAccessToken}`;
```
Comment (pekala) [inline github-temp-token/src/main.ts]: I'd drop the template here, just return the token directly
```suggestion
  return installationAccessToken;
```

File: github-temp-token/src/main.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+};
+
+async function run() {
+  const token = getToken([
+    "ant-man-preview",
+    "machine-man-preview"
+  ]);
```
Comment (pekala) [inline github-temp-token/src/main.ts]: No need for passing those preview flags, they are not used
```suggestion
  const token = getToken();
```

File: github-temp-token/src/main.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import * as core from "@actions/core";
+import { App } from "@octokit/app";
+
+const getToken = (previews?: string[]) => {
```
Comment (pekala) [inline github-temp-token/src/main.ts]:
```suggestion
const getToken = () => {
```

**Pull request merged**